ui/src/main/res/values/styles.xml

```diff
@@ -106,6 +106,10 @@
         <item name="android:textColor">@color/colorSecondaryDark</item>
     </style>
 
+    <style name="HintTextDisabled" parent="TextAppearance.MaterialComponents.Body1">
```

Comment:
how about Widget.Hyperwallet.TextHint.Disabled or Widget.Hyperwallet.Hint.Disabled ?

Reply:
its intentional since we will not share this to global, its used from `Widget.Hyperwallet.TextInputLayout.Disabled`, the same with `ErrorText` and  `HintText`